packages/run-protocol/test/vaultFactory/test-stablecoin.js

```diff
@@ -18,11 +18,11 @@ import {
 import { makePromiseKit } from '@agoric/promise-kit';
 import { makeScriptedPriceAuthority } from '@agoric/zoe/tools/scriptedPriceAuthority.js';
 import { assertAmountsEqual } from '@agoric/zoe/test/zoeTestHelpers.js';
+import { makeParamManagerBuilder } from '@agoric/governance';
```

Comment:
the name of this file should change, yes?

Reply:
good point.  I looked around and found a few files with treasury in their names and renamed them, too.